### PR TITLE
V2 Tests Trigger in V3 Branch

### PR DESCRIPTION
**Pull request by legaspielaine**
TODOs:

- [x] Wait for `version-3` to be merged to `master`
- [x] Remove v3 references in workflows
- [x] Include v2 tests first on the workflows - v3 tests here are not final yet

### CODE REVIEW

File: .github/workflows/e2e-test-wp55.yml [hunk 1]
```diff
@@ -31,9 +31,9 @@ jobs:
 
     - name: Setup the Stackable Free and Premium Repository
       run: |
-        git clone https://github.com/gambitph/Stackable.git
+        git clone -b v3 https://github.com/gambitph/Stackable.git
         cd Stackable
-        git clone https://${{ secrets.ACCESS_KEY }}@github.com/bfintal/Stackable-Premium pro__premium_only
+        git clone -b v3 https://${{ secrets.ACCESS_KEY }}@github.com/bfintal/Stackable-Premium pro__premium_only
```
Comment (legaspielaine) [inline .github/workflows/e2e-test-wp55.yml:36]:
```suggestion
        git clone -b remove-require-context-v3 https://${{ secrets.ACCESS_KEY }}@github.com/bfintal/Stackable-Premium pro__premium_only
```

File: .github/workflows/e2e-test-wp56.yml [hunk 2]
```diff
@@ -31,9 +31,9 @@ jobs:
 
     - name: Setup the Stackable Free and Premium Repository
       run: |
-        git clone https://github.com/gambitph/Stackable.git
+        git clone -b v3 https://github.com/gambitph/Stackable.git
         cd Stackable
-        git clone https://${{ secrets.ACCESS_KEY }}@github.com/bfintal/Stackable-Premium pro__premium_only
+        git clone -b v3 https://${{ secrets.ACCESS_KEY }}@github.com/bfintal/Stackable-Premium pro__premium_only
```
Comment (legaspielaine) [inline .github/workflows/e2e-test-wp56.yml:36]:
```suggestion
        git clone -b remove-require-context-v3 https://${{ secrets.ACCESS_KEY }}@github.com/bfintal/Stackable-Premium pro__premium_only
```

File: .github/workflows/e2e-test-wp57.yml [hunk 3]
```diff
@@ -31,9 +31,9 @@ jobs:
 
     - name: Setup the Stackable Free and Premium Repository
       run: |
-        git clone https://github.com/gambitph/Stackable.git
+        git clone -b v3 https://github.com/gambitph/Stackable.git
         cd Stackable
-        git clone https://${{ secrets.ACCESS_KEY }}@github.com/bfintal/Stackable-Premium pro__premium_only
+        git clone -b v3 https://${{ secrets.ACCESS_KEY }}@github.com/bfintal/Stackable-Premium pro__premium_only
```
Comment (legaspielaine) [inline .github/workflows/e2e-test-wp57.yml:36]:
```suggestion
        git clone -b remove-require-context-v3 https://${{ secrets.ACCESS_KEY }}@github.com/bfintal/Stackable-Premium pro__premium_only
```

File: .github/workflows/e2e-test.yml [hunk 4]
```diff
@@ -31,9 +31,9 @@ jobs:
 
     - name: Setup the Stackable Free and Premium Repository
       run: |
-        git clone https://github.com/gambitph/Stackable.git
+        git clone -b v3 https://github.com/gambitph/Stackable.git
         cd Stackable
-        git clone https://${{ secrets.ACCESS_KEY }}@github.com/bfintal/Stackable-Premium pro__premium_only
+        git clone -b v3 https://${{ secrets.ACCESS_KEY }}@github.com/bfintal/Stackable-Premium pro__premium_only
```
Comment (legaspielaine) [inline .github/workflows/e2e-test.yml:36]:
```suggestion
        git clone -b remove-require-context-v3 https://${{ secrets.ACCESS_KEY }}@github.com/bfintal/Stackable-Premium pro__premium_only
```

**Comment by blankart** [comment]
We can now mere this PR. Change all `v3` references to `master`

**Review by blankart — APPROVED** [review]

**Pull request merged**